lib/commands/devices.js

```diff
@@ -0,0 +1,22 @@
+var firefox = require('node-firefox');
+var common = require('../common');
+
+module.exports = function(program, init) {
+  program.command('devices')
+    .description('List connected devices')
+    .action(init(cmd));
+};
+
+function cmd(options) {
```

Comment:
This is minor, but any reason not to use `command`? It's easier to pick out and may be friendlier to ESL/newer folks not used to having to shorten everything from the DOS days :wink: